src/dash-table/dash/DataTable.js

```diff
@@ -454,34 +420,52 @@ export const propTypes = {
     merge_duplicate_headers: PropTypes.bool,
 
     /**
-     * `n_fixed_columns` will "fix" the set of columns so that
+     * `fixed_columns` will "fix" the set of columns so that
      * they remain visible when scrolling horizontally across
-     * the unfixed columns. `n_fixed_columns` fixes columns
-     * from left-to-right, so `n_fixed_columns=3` will fix
+     * the unfixed columns. `fixed_columns` fixes columns
+     * from left-to-right, so `fixed_columns=3` will fix
```

Comment:
description needs updating

Reply:
Yeah. I know, I'll add a bullet in the description to keep track of it.